db/migrate/20231107235656_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.1]
+  def change
+    create_table :users do |t|
+      t.string :name, null: false
+      t.string :bio, null: false
+      t.integer :posts_counter, default: 0
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
#### Optional

- [ ] Great job on the changes! 👏 I just wanted to give you a heads up that I noticed the user migration table is missing the photo column mentioned in the ERD. It seems like you might have deleted the migration for adding the photo but forgot to update your migrations. No worries, though! 😊 Please take a moment to update your migrations so that the columns specified in the ERD are created in the database. Thanks a bunch! 🌟